### DISCUSSION TITLE
关于2.0开发版的fixed-webview

### DISCUSSION THREAD

**Discussion by 68star89** [libnyanpasu/clash-nyanpasu#1616] [Q&A] [answered]
想请问该类封包相比普通版本修复了哪些内容呢？不太了解fixed-webview和普通版本之间的差异

**Comment by keiko233** [comment, marked as answer]
内嵌了完整的webview，其余无区别

**Comment by qianyueping1211** [comment]
辛苦了，我也没有理解，那什么情况下 使用 webview 版本呢？看两个版本的软件多了200多兆。麻烦了。

**Reply by keiko233** [reply]
系统没有webview的情况可以选这个，非特殊需求不要选这个
